regen/mk_invlists.pl

```diff
@@ -892,17 +892,17 @@ ($$$$$$$)
                 my $code_point = $script_zeros[$i];
                 if (defined $code_point) {
                     $code_point = " 0" if ref $code_point;
-                    print $out_fh "\t$code_point";
+                    print $out_fh "    $code_point";
                 }
                 elsif (lc $enum_list[$i] eq 'inherited') {
-                    print $out_fh "\t 0";
+                    print $out_fh "     0";
```

Comment:
Possibly not for this PR, but that extra space is odd - the effect is that the "0" characters in `0` and `'0'` entries line up, eg in `static const UV script_zeros[]`, which is probably counterproductive.